www_src/pages/section-2/grid.jsx

```diff
@@ -0,0 +1,382 @@
+var React = require('react');
+
+var Slot = require('./slot.jsx');
+var Page = require('./page.jsx');
+var AddPage = require('./add-page.jsx');
```

Comment:
Since this is only added once, does this make sense as a component when it's only a single button with a class?